### ISSUE TITLE
[cocoon] Increased failure rate in adding commits

### ISSUE THREAD

**Issue by CaseyHillers** [flutter/flutter#77385]
![image](https://user-images.githubusercontent.com/2148558/110149681-305d0880-7d93-11eb-8b74-ba9b78c9417d.png)

By default, the GitHub bot throttles commits being merged. In this case, it looks like multiple commits were landed right by each other due to manual merging (should only be done to fix this tree).

The build dashboard did not show this line of failures.

https://github.com/flutter/flutter/commit/8988c4e14c397e890cad816a93fc9dba1582b850 was manually merged, so the previous commit failed to pick up.

This shouldn't happen. Moving to the webhook based approach would get rid of some of the complexity in the existing scheduler as we only need to schedule one commit at a time.

**Comment by CaseyHillers** [comment]
I created plx/script_60._49462f_0000_2e53_920f_089e0832f92c to look at GitHub commits vs Datastore commits.

tl;dr this issue isn't new, and is just an issue with the cron based approach. With the infra team investing more in reliability, we have now only surfaced this issue.

![Screen Shot 2021-03-12 at 2 52 42 PM](https://user-images.githubusercontent.com/2148558/111007004-b274a080-8342-11eb-837e-f5a3dec7aa9a.png)

There is only one or two commits a day that get dropped.

After the webhook approach is landed, we should move the cron based method to an hourly check to validate we have the latest commits from the past day.
